server/models/delius/deliusOffenderManager.ts

```diff
@@ -0,0 +1,9 @@
+export interface DeliusOffenderManager {
+  isResponsibleOfficer: boolean
+  staff: {
```

Comment:
could `staff` itself be missing in the response?

Reply:
I don't think it will ever be `null`, but it could be missing / `undefined` - this was added in the wrong commit but have tidied that up.